soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -19,6 +19,9 @@
 #ifdef CONFIG_BT_SILABS_SIWX91X
 #include "rsi_ble_common_config.h"
 #endif
+#ifdef CONFIG_PM
+#include "sl_si91x_power_manager.h"
+#endif
```

Comment:
Remove `#ifdef` (note that there is `#ifdef` around `rsi_ble_common_config.h` because it will break the compilation otherwise)